src/main/java/com/wine/to/up/deployment/service/controller/ImageController.java

```diff
@@ -21,4 +22,9 @@
     public ImageController(final DeploymentServiceImpl deploymentService) {
         this.deploymentService = deploymentService;
     }
+
+    @GetMapping(value = "/images")
+    public ImageDto images() {
```

Comment:
I'm not sure that API should return our internal DTO. Do you have other options for this one? There is also a problem that it is not obvious what fields would have internal `Map` and there could be problems at front-end with that API

Reply:
I suggest following structure :  `Map<String, Map<String, String>>` , where internal `Map` will store all needed service properties excepting `ID`. It will be used as a key for our main `Map`.

Reply:
Another idea: return `List<ImageVO>`. ImageVO contains all image properties.